app/conf.js

```diff
@@ -42,7 +42,23 @@ const conf = {
   maintenance: {
     flag: process.env.MAINTENANCE_FLAG,
     key: process.env.MAINTENANCE_KEY
-  }
+  },
+  kms: {
+    server: {
+      serviceUrl: process.env.KMS_SERVER_SERVICE_URL || 'https://us-south.kms.cloud.ibm.com',
+      iamAuthUrl: process.env.KMS_SERVER_IAM_AUTH_URL || 'https://iam.cloud.ibm.com',
+      apiKey: process.env.KMS_SERVER_API_KEY,
+      bluemixInstanceGuid: process.env.KMS_SERVER_BLUEMIX_INSTANCE_GUID,
+      rootKeyId: process.env.KMS_SERVER_ROOT_KEY_ID,
+    },
+    cos:{
+      enabled: process.env.KMS_COS_ENABLED,
```

Comment:
Using 'enabled' env vars as booleans always makes me nervous as it's likely someone could try to `export KMS_COS_ENABLED=false` and it would still treat it as true.
```suggestion
      enabled: process.env.KMS_COS_ENABLED && !(process.env.KMS_COS_ENABLED.trim().toLowerCase() === 'false'),
```